bridges/SymfonyCastsBridge.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class SymfonyCastsBridge extends BridgeAbstract {
+	const NAME = 'SymfonyCasts Bridge';
+	const URI = 'https://symfonycasts.com/';
+	const DESCRIPTION = 'Follow new updates on symfonycasts.com';
+	const MAINTAINER = 'No maintainer';
+	const CACHE_TIMEOUT = 3600;
+
+	public function collectData() {
+		$html = getSimpleHTMLDOM('https://symfonycasts.com/updates/find');
```

Comment:
I recommend adding a `returnServerError()` after the `getSimpleHTMLDOM`

```suggestion
		$html = getSimpleHTMLDOM('https://symfonycasts.com/updates/find')
			or returnServerError('Unable to get page.');
```